buildchain/buildchain/codegen.py

```diff
@@ -24,7 +24,7 @@ def codegen_go() -> types.TaskDict:
     """Generate Go code using operator-sdk."""
     cwd  = constants.STORAGE_OPERATOR_ROOT
     actions = []
-    for target in ('k8s', 'openapi'):
+    for target in ('k8s', 'crds'):
```

Comment:
ho, so we don't have to call `openapi-gen` explicitly like Benoit did?

(I thought it was required to update the `zz_generated.openapi.go`)

Reply:
FWIW, IIUC in general you don't need those `zz_generated.openapi.go` files at all.